applications/StructuralMechanicsApplication/structural_mechanics_application.h

```diff
@@ -346,12 +346,12 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) KratosStructuralMechanicsAppl
 
     // Adding the shells elements
     const IsotropicShellElement mIsotropicShellElement3D3N;
-    const ShellThickElement3D4N mShellThickElement3D4N;
-    const ShellThickElement3D4N mShellThickCorotationalElement3D4N;
-    const ShellThinElement3D4N   mShellThinCorotationalElement3D4N;
-    const ShellThinElement3D3N mShellThinElement3D3N;
-    const ShellThinElement3D3N mShellThinCorotationalElement3D3N;
-    const ShellThickElement3D3N  mShellThickCorotationalElement3D3N;
+    const ShellThickElement3D4N<false> mShellThickElement3D4N;
```

Comment:
I would use an enum instead (because otherwise I don't know what is this)

Reply:
how to use an enum for templates?
Do you have an example?

Reply:
~~~c++
enum class Example {ONE, TWO};

template<Example TExample>
void Function()
{
}
...

Function<Example::One>();
~~~

Reply:
Much cleaner, thx for the suggestion, will do it now

Reply:
done, now much better!